src/cpp/rtps/participant/RTPSParticipantImpl.cpp

```diff
@@ -1658,6 +1666,17 @@ bool RTPSParticipantImpl::createSendResources(
     return true;
 }
 
+void RTPSParticipantImpl::setupExternalLocators(
```

Comment:
Please follow naming style rules on new methods.

```suggestion
void RTPSParticipantImpl::setup_external_locators(
```

Reply:
I tried to mimic the style around, but :+1: